bofa_data.php

```diff
@@ -2392,6 +4161,177 @@
 	"weird" => [
 		"+Mysticality substats",
 		"Restore MP (25%)",
-		"a feather (frozen / fetid / flirtatious / flaming / frightful)"
+		[
+			"flaming feather",
+			"frozen feather",
+			"frightful feather",
+			"fetid feather",
+			"flirtatious feather"
+		]
 	]
 ];
+
+$refuse_items = ["big rock",
+"pretty flower",
+"ice-cold sir schlitz",
+"hermit permit",
+"worthless trinket",
+"worthless gewgaw",
+"worthless knick-knack",
+"ice-cold Willer",
+"rusty metal ring",
+"rusty metal shaft",
+"meat from yesterday",
+"spring",
+"sprocket",
+"cog",
+"empty meat tank",
+"ice-cold six-pack",
+"valuable trinket",
+"barbed-wire fence",
+"ghuol egg",
+"skeleton bone",
+"skewer",
+"lihc eye",
+"uncooked chorizo",
+"ice-cold fotie",
+"batgut",
+"briefcase",
+"fat stacks of cash",
+"loose teeth",
+"bat guano",
+"rat appendix",
+"hemp string",
+"gnoll teeth",
+"ten-leaf clover",
+"dead guy's watch",
+"white lightning",
+"mullet wig",
+"tenderizing hammer",
+"linoleum sword hilt",
+"linoleum stick",
+"linoleum crossbow string",
+"asbestos sword hilt",
+"asbestos stick",
+"asbestos crossbow string",
+"chrome sword hilt",
+"chrome stick",
+"chrome crossbow string",
+"yeti fur",
+"penguin skin",
+"yak skin",
+"hippopotamus skin",
+"pirate pelvis",
+"box",
+"bloody clown pants",
+"beer lens",
+"disease",
+"flaming crutch",
+"cast",
+"leather mask",
+"mesh cap",
+"enormous belt buckle",
+"catgut",
+"pr0n legs",
+"pine-fresh air freshener",
+"razor-sharp can lid",
+"Mad Train wine",
+"dirty hobo gloves",
+"furry pants",
+"disturbing fanfic",
+"fruitcake",
+"spiked femur",
+"filthy hippy poncho",
+"broken skull",
+"foon",
+"mob penguin cellular phone",
+"lead necklace",
+"pine tar",
+"tasket",
+"urinal cake",
+"blood flower",
+"lovecat tail",
+"plastic passion fruit",
+"picture of a dead guy's girlfriend",
+"length of string",
+"googly eye",
+"stuffing",
+"felt",
+"wooden block",
+"stench powder",
+"sleaze nuggets",
+"squashed frog",
+"alphabet gum",
+"old leather wallet",
+"old coin purse",
+"gob of wet hair",
+"rock",
+"stringy sinew",
+"stick",
+"tooth",
+"filthy poultice"];
+
+function get_bofa_kill_effect($class_id, $path_id, $monster) {
```

Comment:
not for this PR but i'll put up a PR tomorrow or thursday updating the readme with a human-readable summary of this logic; would love to get your review on it when i finish it. logic makes sense to me but i will definitely want you to validate my description. :-)

Reply:
Sure, ping me and I will check it.